README.md

```diff
@@ -612,6 +613,54 @@ There are 2 options supported for Azure Blob Storage authentication:
 * `manifests_prefix=<prefix>`: set global prefix to store / read manifests on the Azure Blob Storage container (`<container>`) (default: `manifests/`)
 * `name=<manifest>`: name of the manifest to use (default: `buildkit`)
 
+#### Alibaba Cloud OSS cache (experimental)
+
+```bash
+buildctl build ... \
+  --output type=image,name=docker.io/username/image,push=true \
+  --export-cache type=oss,endpoint_url=https://oss-cn-hangzhou.aliyuncs.com,bucket=my_bucket,name=my_image \
+  --import-cache type=oss,endpoint_url=https://oss-cn-hangzhou.aliyuncs.com,bucket=my_bucket,name=my_image
+```
+
+The following attributes are required:
+* `bucket`: Alibaba Cloud OSS bucket (default: `$ALIBABA_CLOUD_OSS_BUCKET`)
+* `endpoint_url`: specify a specific OSS endpoint (default: empty)
+
+Storage locations:
+* blobs: `oss://<bucket>/<prefix><blobs_prefix>/<sha256>`, default: `oss://<bucket>/blobs/<sha256>`
+* manifests: `oss://<bucket>/<prefix><manifests_prefix>/<name>`, default: `oss://<bucket>/manifests/<name>`
+
+S3 configuration:
+* `blobs_prefix`: global prefix to store / read blobs on OSS (default: `blobs/`)
+* `manifests_prefix`: global prefix to store / read manifests on OSS (default: `manifests/`)
+
+Alibaba Cloud Authentication:
+
+The simplest way is to use a pair of AK/SK:
+
+* Any system using environment variables / config files supported by the [Alibaba Cloud OSS Go SDK](https://www.alibabacloud.com/help/en/oss/developer-reference/go/). The configuration must be available for the buildkit daemon, not for the client.
+* Using the following attributes:
+  * `access_key_id`: Access Key ID
+  * `secret_access_key`: Secret Access Key
+  * `secret_access_key`: Security Token (optional)
+
+`--export-cache` options:
+* `type=oss`
```

Comment:
I think this should be called "alibaba-oss" to avoid confusion

Reply:
I also wonder if this should be an external plugin that would talk to buildkitd over grpc or something